pre-scan.pl

```diff
@@ -284,9 +284,8 @@
 } else {
 	&gone ("Tried to release lock on $lockFile, but it didn't exist!");
 }
-#
-# (Commvault performs the backup)
-#
-# NOTE Post job will unmount the snapshot, and delete the snapshot, in just a few minutes
+
+# (External backup software now performs the backup)
```

Comment:
Would it make sense to name Veeam as the backup software we will be using? Another way to frame the question is, will the script be completely agnostic of the backup solution?

Reply:
No, because Veeam is already in the name of the repo, and in the frontmatter comments.  
I specifically wanted to reduce the number of times I used the word "Veeam", so that the next time this is ported, it's less work.